dist/docker/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# This Dockerfile describes a base image that includes nodejs, python3,
+# as well as the cli for the major clour providers Pulumi supports
+# (aws, azure, gcp, kubernetes) as well as the Pulumi CLI itself.
+#
+# It is suitable as a general base image for an environment for deploying
+# infrastructure written with Pulumi.
+FROM debian:stretch
+
+ARG PULUMI_VERSION
+
+# Install some runtime pre-reqs.
+RUN apt-get update -y
+RUN apt-get install -y ca-certificates curl software-properties-common gnupg jq git
+
+# Install the Pulumi SDK, including the CLI and language runtimes.
+RUN curl -fsSL https://get.pulumi.com/ | bash -s -- --version $(echo $PULUMI_VERSION | cut -c 2-) && \
```

Comment:
Should this default to "latest" if built without a `PULUMI_VERSION` argument? It would be nice if `docker build .` would just work in the repo.

Reply:
That makes sense.